## 1.Faça a análise descritiva das variáveis X1 a X7, X9 e X10

In [1]:
# Importar bibliotecas necessárias
import pandas as pd

In [2]:
# Local do arquivo
caminho = '/Users/fabicampanari/Desktop/_8-Prova Matematematica/1-statiscalMeasures_ Hypothesis Testing II./1🇧🇷-statiscalMeasures_ Hypothesis Testing II./MATEMA/Biscobis.xlsx'

In [3]:
df=pd.read_excel(caminho)

In [4]:
# Visualizar as primeiras linhas do DataFrame para entender a estrutura dos dados
print("Primeiras linhas do DataFrame:")
print(df.head())

Primeiras linhas do DataFrame:
   ident   x1   x2   x3   x4   x5   x6   x7  x8  x9  x10  x11  x12  x13  x14
0      1  4.1  0.6  6.9  4.7  2.4  2.3  5.2   0  32  4.2    1    0    1    1
1      5  6.0  0.9  9.6  7.8  3.4  4.6  4.5   0  58  6.8    1    0    1    3
2      7  4.6  2.4  9.5  6.6  3.5  4.5  7.6   0  46  5.8    1    0    1    1
3      9  5.5  1.6  9.4  4.7  3.5  3.0  7.6   0  63  5.4    1    0    1    3
4     11  2.4  1.6  8.8  4.8  2.0  2.8  5.8   0  32  4.3    1    0    0    1


In [5]:
# Selecionar as variáveis de interesse (X1 a X7, X9 e X10)
variaveis_interesse = ['x1', 'x2', 'x3', 'x4', 'x5', 'x6', 'x7', 'x9', 'x10']
df_selecionado = df[variaveis_interesse]

In [6]:
# Realizar a análise descritiva das variáveis selecionadas
analise_descritiva = df_selecionado.describe()
print("Análise Descritiva das Variáveis X1 a X7, X9 e X10:")
print(analise_descritiva)

Análise Descritiva das Variáveis X1 a X7, X9 e X10:
               x1          x2          x3          x4          x5          x6  \
count  100.000000  100.000000  100.000000  100.000000  100.000000  100.000000   
mean     3.515000    2.364000    7.894000    5.258000    2.913000    2.665000   
std      1.320726    1.195659    1.386502    1.130047    0.750307    0.770855   
min      0.000000    0.200000    5.000000    2.500000    0.700000    1.100000   
25%      2.500000    1.475000    6.700000    4.575000    2.400000    2.200000   
50%      3.400000    2.150000    8.050000    5.000000    3.000000    2.600000   
75%      4.600000    3.225000    9.100000    6.000000    3.425000    3.000000   
max      6.100000    5.400000   10.000000    8.200000    4.600000    4.600000   

               x7         x9         x10  
count  100.000000  100.00000  100.000000  
mean     6.957000   46.10000    4.771000  
std      1.590645    8.98877    0.855558  
min      3.700000   25.00000    3.200000  
25%

## Interpretação e Conclusão

### Resultados da Análise Descritiva:

### Resultados da Análise Descritiva:

Análise Descritiva das Variáveis X1 a X7, X9 e X10:

|       |    x1    |    x2    |    x3    |    x4    |    x5    |    x6    |    x7    |    x9    |   x10   |
|-------|----------|----------|----------|----------|----------|----------|----------|----------|---------|
| count | 100.0000 | 100.0000 | 100.0000 | 100.0000 | 100.0000 | 100.0000 | 100.0000 | 100.0000 | 100.0000 |
| mean  |   3.5150 |   2.3640 |   7.8940 |   5.2580 |   2.9130 |   2.6650 |   6.9570 |  46.1000 |   4.7710 |
| std   |   1.3207 |   1.1957 |   1.3865 |   1.1300 |   0.7503 |   0.7709 |   1.5906 |   8.9888 |   0.8556 |
| min   |   0.0000 |   0.2000 |   5.0000 |   2.5000 |   0.7000 |   1.1000 |   3.7000 |  25.0000 |   3.2000 |
| 25%   |   2.5000 |   1.4750 |   6.7000 |   4.5750 |   2.4000 |   2.2000 |   5.8000 |  39.0000 |   4.1000 |
| 50%   |   3.4000 |   2.1500 |   8.0500 |   5.0000 |   3.0000 |   2.6000 |   7.1500 |  46.5000 |   4.8500 |
| 75%   |   4.6000 |   3.2250 |   9.1000 |   6.0000 |   3.4250 |   3.0000 |   8.3250 |  53.2500 |   5.4000 |
| max   |   6.1000 |   5.4000 |  10.0000 |   8.2000 |   4.6000 |   4.6000 |  10.0000 |  65.0000 |   6.8000 |


Interpretação:
**Contagem (count):** Número de observações para cada variável.

**Média (mean):** Valor médio de cada variável.

**Desvio Padrão (std):** Medida da dispersão dos valores em torno da média.

**Mínimo (min):** Valor mínimo observado para cada variável.

**1º Quartil (25%):** Valor abaixo do qual 25% das observações caem.

**Mediana (50%):** Valor central das observações.

**3º Quartil (75%):** Valor abaixo do qual 75% das observações caem.

**Máximo (max):** Valor máximo observado para cada variável.

<br>

**Conclusão:**

A análise descritiva das variáveis X1 a X7, X9 e X10 fornece uma visão geral das características estatísticas dessas variáveis na base de dados BISCOBIS. Com base nos resultados, podemos observar a distribuição dos valores, a centralidade (média e mediana) e a dispersão (desvio padrão) das variáveis. Esses insights são úteis para entender melhor os dados e identificar possíveis padrões ou anomalias.

